src/js/main.js

```diff
@@ -25,3 +25,27 @@ document.addEventListener("click", (event) => {
 document.querySelector(".site-nav .nav-smallScreen .search-container svg").addEventListener("click", () => {
 	document.querySelector(".site-nav .nav-smallScreen .search-container input").focus();
 });
+
+// Configuration for observer window size/location in viewport
+// Reference: https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API#Creating_an_intersection_observer
+const observerOptions = {
+	rootMargin: "0% 0% -80% 0%"
```

Comment:
Can you add a code comment to explain the purpose of this line - the observer action kicks in when the 80% of the target elements become visible?